atr/post/draft.py

```diff
@@ -37,37 +36,12 @@
 import atr.web as web
 
 
-class VotePreviewForm(forms.Typed):
-    body = forms.textarea("Body")
-    # TODO: Validate the vote duration again?
```

Comment:
This comment was lost in the new class. Is the comment, or any of its three lines, important? Should it have been retained, in part or in whole?